### PR TITLE
Add topic "localstorage"

### PR DESCRIPTION
**Pull request by shaedrich**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created the topic or collection I'm changing.

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

LocalStorage is similarily important as cookies

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by shaedrich** [comment]
Or should this better be [WebStorage](https://github.com/topics/webstorage) and include LocalStorage and [SessionStorage](https://github.com/topics/sessionstorage)?

File: topics/localstorage/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+aliases: local-storage
+display_name: LocalStorage
+short_description: LocalStorage is a JavaScript API of storing arbitrary key-value data in the web browser as an alternative to cookies.
```
Comment (zkoppert) [inline topics/localstorage/index.md:4]:
```suggestion
short_description: LocalStorage is a JavaScript object that allows storing arbitrary key-value pairs in the web browser as an alternative to cookies.
```

File: topics/localstorage/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+topic: localstorage
+url: https://html.spec.whatwg.org/multipage/webstorage.html#dom-localstorage-dev
+wikipedia_url: https://en.wikipedia.org/wiki/Web_storage
+related: sessionstorage, indexeddb, websql, web-storage, webstorage
+---
+LocalStorage is a JavaScript API of storing arbitrary key-value data in the web browser as an alternative to cookies&mdash;another alterantive if the data in question goes beyond key-value pairs is to use the IndexedDB.
```
Comment (zkoppert) [inline topics/localstorage/index.md:10]:
```suggestion
LocalStorage is a JavaScript object that allows storing arbitrary key-value pairs in the web browser as an alternative to cookies&mdash;another alternative if the data in question goes beyond key-value pairs is to use the IndexedDB.
```

**Review by tomthorogood — APPROVED** [review]
Thanks for the contribution!

**Comment by shaedrich** [comment]
You're welcome!

**Pull request merged**